utils/utils.py

```diff
@@ -149,7 +149,7 @@ def wrap_env(env):
         return None
 
 
-def make_env(env_id, rank=0, seed=0, log_dir=None, wrapper_class=None):
+def make_env(env_id, rank=0, seed=0, log_dir=None, wrapper_class=None, env_kwargs={}):
```

Comment:
You should never use a mutable as default argument or you will get [weird behavior](https://github.com/satwikkansal/wtfpython#-beware-of-default-mutable-arguments).
please do:
```python
make_env(..., env_kwargs=None)
....
if env_kwargs is None:
    env_kwargs = {}
```

Reply:
Good to know that! I'll make the changes.

Reply:
I've made the suggested change.